pkg/plugins/k8s/k8s_plugin.go

```diff
@@ -127,6 +129,14 @@ func (p *K8sPlugin) OnNodeStateChange(old, new *sriovnetworkv1.SriovNetworkNodeS
 	needDrain = false
 	needReboot = false
 
+	if p.loadedManifests != loaded {
```

Comment:
to avoid this, you could define a `New<plugin-name>Plugin()` method for the plugins and use it in `plugins.go`

in this method you can perform one-time initialization.